lint-configs/package.json

```diff
@@ -22,9 +22,7 @@
     "./template-lint": {
       "require": "./template-lint.config.cjs"
     },
-    "./eslint-discourse/*": {
```

Comment:
The `eslint-discourse/` export was committed mistakenly as part of the custom eslint rule work. It's unused, and didn't even resolve to any files.